src/layman/authz/role_service.py

```diff
@@ -22,3 +22,9 @@ def ensure_admin_roles():
     select %s, 'ADMIN'
     ;"""
     db_util.run_statement(create_admin_user_roles_view, (settings.LAYMAN_GS_USER, settings.LAYMAN_GS_ROLE, settings.LAYMAN_GS_USER))
+
+
+def get_user_roles(username):
```

Comment:
filter out admin and user roles

Reply:
filter out also roles not matching new role regexp pattern (see WORKSPACE_NAME_PATTERN)